conf2sql.pl

```diff
@@ -29,16 +29,23 @@
 
 die "$program: missing --dbdriver option\n" unless defined($opts{'dbdriver'});
 die "$program: missing --dbname option\n" unless defined($opts{'dbname'});
+if (!$opts{'dbdriver'} =~ /sqlite/i) {
```

Comment:
i'm testing your module with sqlite, and i'm correcting each problem i found.
the first was in conf2sql.pl to support sqlite!
this modification make possible tu use sqlite with this script.
the use is: ./conf2sql.pl --dbdriver=SQLite --dbname=/path/to/sqlite.db